src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -37,8 +41,8 @@ ColorScheme _CreateCampbellScheme()
                                  RGB(12, 12, 12) };
     auto& campbellTable = campbellScheme.GetTable();
     auto campbellSpan = gsl::span<COLORREF>(&campbellTable[0], gsl::narrow<ptrdiff_t>(COLOR_TABLE_SIZE));
-    Microsoft::Console::Utils::InitializeCampbellColorTable(campbellSpan);
-    Microsoft::Console::Utils::SetColorTableAlpha(campbellSpan, 0xff);
+    Utils::InitializeCampbellColorTable(campbellSpan);
+    Utils::SetColorTableAlpha(campbellSpan, 0xff);

```

Comment:
0xFF is being used a bunch, can it be a named constant?

Reply:
`BYTE_MAX`

Reply:
not in this code review.

Reply:
BYTE_MAX is a functional constant, but not a semantically correct one ..

Reply:
`MOST_COLOR`